package vtrc_rpc_lowlevel;


// index 51100

message lowlevel_unit {


    // --- //
    optional bytes request  = 20;
    optional bytes response = 21;
}
